src/utils/apiClient.ts

```diff
@@ -9,3 +10,23 @@ const apiClient = axios.create({
 });
 
 export default apiClient;
+
+export const postRequest = async (url: string, data: any) => {
+  const token = Cookies.get('jwt');
+  try {
+    const response = await apiClient.post(url, data, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    return response;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const formDataEntries = (event: React.FormEvent<HTMLFormElement>) => {
+  const formData = new FormData(event.target as HTMLFormElement);
+  const data = Object.fromEntries(formData);
+  return data;
```

Comment:
```suggestion
return Object.formEntries(formData);
```
좀 더 간결하게 oneline return은 어떨까요